src/components/Involvements/index.scss

```diff
@@ -1,6 +1,6 @@
-@import '../../../../vars';
+@import '../../vars';
 
-.myprofile-involvements {
+.involvements {
```

Comment:
Correct. Before, the involvements folder was located in the My Profile Component but I moved it so that it's in "src/Components" instead. I moved it so that both the My Profile and Public Profile Component can reference it. Since the folder has been relocated, I had to change the CSS style names since it still showed "myprofile"